node/service/src/chain_spec.rs

```diff
@@ -1560,6 +1569,49 @@ pub fn westend_development_config() -> Result<WestendChainSpec, String> {
 	))
 }
 
+/// Rococo development config (single validator Alice)
+pub fn rococo_development_config() -> Result<RococoChainSpec, String> {
+	let wasm_binary = rococo::WASM_BINARY.ok_or("Rococo development wasm not available")?;
+
+	Ok(RococoChainSpec::from_genesis(
+		"Development",
+		"rococo_dev",
+		ChainType::Development,
+		move || RococoGenesisExt {
+			runtime_genesis_config: rococo_development_config_genesis(wasm_binary),
+			// Use 1 minute session length.
+			session_length_in_blocks: Some(10),
+		},
+		vec![],
+		None,
+		Some(DEFAULT_PROTOCOL_ID),
+		None,
+		Default::default(),
+	))
+}
+
+/// Wococo development config (single validator Alice)
+pub fn wococo_development_config() -> Result<RococoChainSpec, String> {
+	const WOCOCO_DEV_PROTOCOL_ID: &str = "woco";
```

Comment:
I added this since having both a Rococo-Dev and Wococo-Dev node running locally would cause them to peer with each other - open to other suggestions for avoiding this